### PR TITLE
Auth: Support list and revoke of user auth tokens in UI

### PR DESCRIPTION
**Pull request by shavonn**
Feature (#16223)
Dependent on https://github.com/grafana/grafana/pull/17504

### CODE REVIEW

**Comment by torkelo** [comment]
great job on this! 

Will try to decide soon if we merge this and then do the refactor (to use same component in profile & admin) & react conversion or we keep this PR open as a start and the the next refactoring & react conversions in this PR.

File: public/app/core/utils/userAgent.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function getBrowserOS(userAgent: string): string {
```
Comment (marefr) [inline public/app/core/utils/userAgent.ts]: There is a related issue #16222 to implement useragent parsing in the HTTP API that would be the proper way forward (I think). If you want to merge this PR as is before iterating further that's fine as a temporary thing.

Comment (shavonn) [reply public/app/core/utils/userAgent.ts]: Thanks for pointing that issue out, Marcus. Will grab it as well.

**Comment by shavonn** [comment]
Depends on https://github.com/grafana/grafana/pull/17504

File: public/app/features/admin/AdminEditUserCtrl.ts [hunk 2]
```diff
@@ -1,19 +1,23 @@
 import _ from 'lodash';
+import { dateTime } from '@grafana/ui/src/utils/moment_wrapper';
```
Comment (dprokop) [inline public/app/features/admin/AdminEditUserCtrl.ts]: You don't need a full path to import `dataTime`, just import from `@grafana/ui` :)

File: public/app/features/profile/ProfileCtrl.ts [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import config from 'app/core/config';
 import { coreModule } from 'app/core/core';
+import { dateTime } from '@grafana/ui/src/utils/moment_wrapper';
```
Comment (dprokop) [inline public/app/features/profile/ProfileCtrl.ts]: Same as my previous comment

File: public/app/features/profile/ProfileCtrl.ts [hunk 4]
```diff
@@ -27,6 +31,51 @@ export class ProfileCtrl {
     });
   }
 
+  getUserSessions() {
+    this.backendSrv.get('/api/user/auth-tokens').then((sessions: UserSession[]) => {
+      console.log(sessions);
```
Comment (dprokop) [inline public/app/features/profile/ProfileCtrl.ts]:
```suggestion

```

**Review by dprokop — APPROVED** [review]
LGTM

**Pull request merged**